containerd-shim-spin/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "containerd-shim-spin-v2"
-version = "0.10.0"
+version = "0.12.0"
 authors = ["DeisLabs Engineering Team"]
```

Comment:
Should we call this the "SpinKube Engineering Team" now?

Reply:
done